node/src/components/consensus/highway_core/state.rs

```diff
@@ -774,6 +774,24 @@ impl<C: Context> State<C> {
         self.units.is_empty()
     }
 
+    /// Returns the set of units (by hash) that are endorsed and seen from the panorama.
+    #[allow(unused)] // TODO: Use for creating new LNC-compliant units.
+    fn seen_endorsed(&self, pan: &Panorama<C>) -> BTreeSet<C::Hash> {
```

Comment:
Why not call it already? This is a feature branch anyway, so it won't break `master` when merged. It will also allow Matt to start writing unit tests.

Reply:
Done.